--- st/config.def.h	2024-01-21 18:22:09.366805939 +0900
+++ config.def.h	2024-01-21 18:25:10.326989445 +0900
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Noto Sans Mono CJK JP:pixelsize=16:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -141,7 +141,7 @@
  * 6: Bar ("|")
  * 7: Snowman ("☃")
  */
-static unsigned int cursorshape = 2;
+static unsigned int cursorshape = 6;
 
 /*
  * Default columns and rows numbers
